package.json

```diff
@@ -126,10 +141,20 @@
           "type": "number",
           "default": 20
         },
+        "doxdocgen.generic.authorName": {
+          "description": "Set the name of the author.  Replaces {author}.",
+          "type": "string",
+          "default": "your name"
+        },
+        "doxdocgen.generic.authorEmail": {
+          "description": "Set the e-mail address of the author.  Replaces {email}.",
+          "type": "string",
+          "default": "you@domain.com"
+        },
         "doxdocgen.generic.authorTag": {
           "description": "Set the style of the author tag and your name.",
           "type": "string",
-          "default": "@author your name"
+          "default": "@author {author} ({email})"
```

Comment:
Please use a new tag for this. I don't want it to break existing configs.

Reply:
Does it break existing configs?  Users can always hardcode their name in the authorTag, rather than using the {author}/{email} templates.

Reply:
Yeah you're right. Just add it to the description which tags are supported.